teams/teams-ps/teams/Get-CsOnlineTelephoneNumberOrder.md

```diff
@@ -13,82 +13,158 @@ ms.reviewer: julienp
+
 
 ## EXAMPLES
 
-### -------------------------- Example 1 --------------------------
+### Example 1
 ```
-PS C:\> $order = Get-CsOnlineTelephoneNumberOrder -OrderId 1efd85ca-dd46-41b3-80a0-2e4c5f87c912
-
-AreaCode              :
-CivicAddressId        :
-CountryCode           : US
-CreatedAt             : 8/23/2021 5:43:44 PM
-Description           : test
-ErrorCode             : NoError
-Id                    : 1efd85ca-dd46-41b3-80a0-2e4c5f87c912
-InventoryType         : Subscriber
-IsManual              : False
-Name                  : test
-NumberPrefix          : 1718
-NumberType            : UserSubscriber
-Quantity              : 1
-ReservationExpiryDate : 8/23/2021 5:59:45 PM
-SearchType            : Prefix
-SendToServiceDesk     : False
-Status                : Reserved
-TelephoneNumber       : {Microsoft.Teams.ConfigAPI.Cmdlets.Generated.Models.TelephoneNumberSearchResult}
-
-PS C:\> $order.TelephoneNumber
-
-Location      TelephoneNumber
---------      ---------------
-New York City +17182000004
+PS C:\> $order = Get-CsOnlineTelephoneNumberOrder -OrderType Search -OrderId 1fd52b3b-b804-4ac4-a84d-4d70b51dd4be
+
+Key                   Value
+---                   -----
+Id                    1fd52b3b-b804-4ac4-a84d-4d70b51dd4be
+Name                  Postal Code Search Test
+CreatedAt             2024-11-30T00:34:00.0825627+00:00
+CreatedBy             ContosoAdmin
+Description           Postal Code Search Test
+NumberType            UserSubscriber
+SearchType            PostalCode
+AreaCode              778
+PostalOrZipCode       V7Y 1G5
+Quantity              2
+Status                Reserved
+IsManual              False
+TelephoneNumbers      {System.Collections.Generic.Dictionary`2[System.String,System.Object], System.Collections.Generic.Dictionary`2[System.String,System.Object]}
+ReservationExpiryDate 2024-11-30T00:50:01.1794152+00:00
+ErrorCode             NoError
+InventoryType         Subscriber
+SendToServiceDesk     False
+CountryCode           CA
+ 
+PS C:\> $order.TelephoneNumbers
+ 
+Key             Value
+---             -----
+Location        Vancouver
+TelephoneNumber +16046606034
+Location        Vancouver
+TelephoneNumber +16046606030
 ```
 
-This example returns a successful telephone number search and the telephone number +17182000004 is reserved for purchase.
+This example returns a successful telephone number search and the telephone numbers are reserved for purchase.
 
-### -------------------------- Example 2 --------------------------
+### Example 2
 ```
 PS C:\> $order = Get-CsOnlineTelephoneNumberOrder -OrderId 8d23e073-bc98-4f73-8e05-7517655d7042
```

Comment:
Nit: Suggest to add -OrderType to this argument because we're covering the skipping of -OrderType in Example 3; Example 2 is focused on the error search order status
`PS C:\> $order = Get-CsOnlineTelephoneNumberOrder -OrderType Search -OrderId 8d23e073-bc98-4f73-8e05-7517655d7042`

Reply:
added